static/app/gettingStartedDocs/apple/ios.tsx

```diff
@@ -713,7 +716,12 @@ const replayOnboarding: OnboardingConfig<PlatformOptions> = {
       ],
     },
   ],
-  verify: () => [],
+  verify: getReplayVerifyStep({
+    replayOnErrorSampleRateName:
+      'options\u200b.experimental\u200b.sessionReplay\u200b.onErrorSampleRate',
```

Comment:
Added zero width spaces here to allow wrapping lines at each segment. Otherwise it messes up the layout.